util/processDashBoardData.test.js

```diff
@@ -0,0 +1,98 @@
+import { processDashboardData } from './processDashBoardData';
```

Comment:
Rename this import to fit the new name of your file (`util/processDashboardData.js`) Otherwise, the test will not pass.

Reply:
fixed